roles/StackStorm.st2smoketests/defaults/main.yml

```diff
@@ -18,6 +18,6 @@ ewc_smoke_tests_rbac:
             - "runner_type_list"
 
   assignments:
-    - name: "{{ ewc_smoke_tests_user }}"
+    - name: "{{ smoke_tests_rbac_user }}"
       roles:
-        - ewc_smoke_tests_basic
+        - smoke_tests_rbac_basic
```

Comment:
Please include the newline at the end of this and other files where it was removed.
See https://stackoverflow.com/a/729795/4533625 for more info.

Reply:
Done, newlines were added. Will look into the tests tomorrow.